### DISCUSSION TITLE
Sending Power level to the API

### DISCUSSION THREAD

**Discussion by Michael-AUG** [magicbug/Cloudlog#2163] [Q&A]
Hi
I'm trying to fiddle around with a PHP script to upload VFO details to Cloudlog. I've managed to get the VFO and mode uploaded, but can't work out how to get the Power level included.
Could anyone advise?
Many thanks
Mike GM5AUG

**Comment by phl0** [comment]
Should be as easy as asending all the other stuff. Include a 
```
"power" : "10" 
```
to the JSON sent for example. See https://github.com/myzinsky/CloudLogCatQt/pull/6 as my patch that added power to CloudLogCatQt for another example.
